### ISSUE TITLE
[SAGE] Example Collection

### ISSUE THREAD

**Issue by devangm** [TAMULib/SAGE#564]
**URL:** https://library.tamu.edu/discovery/discovery-context/berger-cloonan?direction=ASC
**Violation:** A: Info and Relationships
**Issues:** Select missing label
**Notes:** Title' & 'No. per page' select fields are missing labels in top and bottom of page

**URL:** https://library.tamu.edu/discovery/discovery-context/berger-cloonan?direction=ASC
**Violation:** A: Info and Relationships
**Issues:** Missing first level heading
**Notes:**  'h1' is not present on page

**URL:** https://library.tamu.edu/discovery/discovery-context/berger-cloonan?direction=ASC
**Violation:**  A: Link Purpose (In Context) 
**Issues:** Redundant Link
**Notes:**  Image and Title of item link to same URL. Image does not need to have link (image missing alt text & img descripion   wrong) or read by NVDA / combine to one link

**URL:**  https://library.tamu.edu/discovery/discovery-context/berger-cloonan?direction=ASC
**Violation:**  AA: Contrast (Minimum)
**Issues:** Very low contrast
**Notes:**  Under heading, breadcrumb link text is very light

**URL:**  https://library.tamu.edu/discovery/discovery-context/berger-cloonan?direction=ASC
**Violation:**  A:Bypass Blocks
**Issues:** Empty heading
**Notes:**  Each item has a heading that is empty

**Comment by qtamu** [comment]
Change breadcrumb color
